test/harmony-regression/HarmonyRegression.ipynb

```diff
@@ -85,7 +85,7 @@
    "source": [
     "### Sample Sync Request - Bounding Box and Temporal Subsetting with Reformatting\n",
     "\n",
-    "Harmony/gdal only working in UAT due to Harmony-600."
+    "harmony/service-example only working in UAT due to Harmony-600."
    ]
```

Comment:
Should it be 'harmonyservices/service-example'?

Reply:
Nope. I checked the versions endpoint for the name.